base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -222,6 +223,32 @@ Result<StatusFetcherOutput> StatusFetcher::FetchStatus(
   };
 }
 
+Result<Json::Value> StatusFetcher::FetchGroupStatus(
+    const selector::LocalInstanceGroup& group,
+    const RequestWithStdio& original_request) {
+  Json::Value group_json(Json::objectValue);
+  group_json["group_name"] = group.GroupName();
+  group_json["start_time"] = selector::Format(group.StartTime());
+
+  auto request_message = MakeRequest(
+      {.cmd_args = {"cvd", "status", "--print", "--all_instances"},
+       .env = cvd_common::ConvertToEnvs(
+           original_request.Message().command_request().env()),
+       .selector_args = {"--group_name", group.GroupName()},
+       .working_dir =
+           original_request.Message().command_request().working_directory()});
+  RequestWithStdio group_request{request_message,
+                                 original_request.FileDescriptors()};
+  auto [_, instances_json, group_response] =
+      CF_EXPECT(FetchStatus(group_request));
+  CF_EXPECT_EQ(
+      group_response.status().code(), cvd::Status::OK,
+      fmt::format("Running cvd status --all_instances for group \"{}\" failed",
```

Comment:
nit: put the  `cvd status --all_instances` part between backquotes to have the command part easier to identify.

Reply:
Done